### ISSUE TITLE
Is there any unnecessary coupling?

### ISSUE THREAD

**Issue by samreid** [phetsims/energy-skate-park#229]
From #202 I searched for "model, " and found occurrences that may be over-coupled, including:

* new Track(model)
* new ToolboxPanel(model,view)
* ControlPointUI(model)
* new EnergyBarGraphAccordionBox( model,
* new VisibilityControlsPanel( model,

**Comment by jessegreenberg** [comment]
I reviewed the classes mentioned, some of them could be decoupled but most seemed best as they are.
- Track: Actually, several of the args to this constructor can be found on the model anyway. And three other model fields were used in this class. So I removed these args from the constructor.
- ToolboxPanel: The class uses 4 model fields and 3 view fields. Decoupling would result in a constructor with 9 args. I feel like the current set up is best.
- ControlPointUI:  Class uses 3 model functions, I think it is best to pass the model in to use these.
- EnergyBarGraphAccordionBox - uses 3 model fields, this seemed like a good place to decouple, done in the above commits.
- VisibilityControlsPanel -> model is passed to EnergySkateParkVisibilityControls, which uses many Properties on the model to control visibility of various things. Passing in model seems best.

Back to @samreid to review.
